raster3d/raster3dintro.html

```diff
@@ -174,17 +174,17 @@ <h3>Working with 3D visualization software</h3>
 GRASS GIS 3D raster maps can be exported to VTK
 using <a href="r3.out.vtk.html">r3.out.vtk</a>.
 VTK files can be visualized with the
-<em><a href="http://www.vtk.org">VTK Toolkit</a></em>,
-<em><a href="http://www.paraview.org">Paraview</a></em> and
-<em><a href="http://mayavi.sourceforge.net">MayaVi</a></em>.
+<em><a href="https://vtk.org">VTK Toolkit</a></em>,
+<em><a href="https://www.paraview.org">Paraview</a></em> and
+<em><a href="https://mayavi.sourceforge.net">MayaVi</a></em>.
```

Comment:
This mayavi is for the really older mayavi 1. There was a complete rewrite named mayavi2, but now the current one is named only mayavi, they are at version 4.7.3 and the repo is https://github.com/enthought/mayavi